language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/definitions/FunctionDefinitionCompletionTest.java

```diff
@@ -134,6 +134,7 @@ public void test(String config, String configPath) throws WorkspaceDocumentExcep
 //                {"ifWhileConditionContextCompletion3.json", "function"},
```

Comment:
is this an unnecessary test ?

Reply:
This was disabled earlier due to a source pruner bug.